### ISSUE TITLE
Breaking change from `0.9.1` -> `0.9.3`

### ISSUE THREAD

**Issue by torokati44** [zbraniecki/unic-locale#75]
Following https://github.com/zbraniecki/unic-locale/issues/74, there is still something funky going on with `langid!`:

https://github.com/torokati44/ruffle-android/actions/runs/7147238350/job/19466398830

```
13 | pub static US_ENGLISH: LanguageIdentifier = langid!("en-US");
   |                                             ^^^^^^^^^^^^^^^^
   |
   = help: message: Failed to find the crate in Cargo.toml: NotFound
   ```

**Comment by zbraniecki** [comment]
Ugh, so this is a case of  a deep nested dependency. I don't know how to replace the `proc-macro-hack` concept of `$crate`. I'll try to debug it over the next days.

If it starts affecting people more, I'll yank 0.9.3 as well tomorrow.

**Comment by zbraniecki** [comment]
I yanked 0.9.3 and published 0.9.4 which uses proc_macro_hack again. I think it fixes your case, please confirm.

**Comment by torokati44** [comment]
Can confirm, 0.9.4 works fine. Thank you!
